add-on/chrome/bootstrap.js

```diff
@@ -760,25 +760,6 @@ var WindowListener = {
           return;
         }
 
-        let buckets = this.constants.SHARING_SCREEN;
```

Comment:
It looks like all of this function can be removed, and the `_pauseButtonClicked`/`_resumeButtonClicked` member variables.